src/editables/redirector.py

```diff
@@ -32,11 +32,7 @@ def find_spec(
 
     @classmethod
     def install(cls) -> None:
-        for f in sys.meta_path:
-            if f == cls:
-                break
-        else:
-            sys.meta_path.append(cls)
+        cls in sys.meta_path or sys.meta_path.append(cls)  # type: ignore[func-returns-value]
```

Comment:
While I don't like the "pure expression" form here, I will concede that the "break-else" approach is unnecessary optimisation, as `sys.meta_path` is never going to be long enough that stopping early matters. The following is much simpler:

```
if cls not in sys.meta_path:
    sys.meta_path.append(cls)
```

I'll make this change.

Reply:
I was going to suggest that alternative as well, so thanks.

Another way you could consider doing it:

```python
sys.meta_path.extend([cls] * (cls not in sys.meta_path))
```

or

```python
sys.meta_path.extend(filterfalse(sys.meta_path.__contains__, [cls]))
```

But I agree, the two-line if statement is probably the most idiomatic.